airflow/www/views.py

```diff
@@ -2099,7 +2099,7 @@ def on_model_change(self, form, model, is_created):
 
     @classmethod
     def alert_fernet_key(cls):
-        return not configuration.has_option('core', 'fernet_key')
+        return conf.get('core', 'fernet_key') is None
```

Comment:
This fix addresses the problem raised by the reporter in https://github.com/airbnb/airflow/issues/920, but I've raised some concerns about how we are managing configuration in that issue, which we can fix by providing an easy way to inspect a unified view of the winning configuration in one place.